app/src/main/java/com/kylehanish/todo/interfaces/iTodoItemChangeListener.java

```diff
@@ -0,0 +1,20 @@
+package com.kylehanish.todo.interfaces;
+
+import android.view.View;
+
+import com.kylehanish.todo.classes.TodoItem;
+
+/**
+ * Created by Kyle Hanish on 4/29/18.
+ */
+
+public interface iTodoItemChangeListener {
```

Comment:
Nice use of an interface to encapsulate this functionality.

Reply:
Agreed